internal/namespaces/k8s/v1beta4/custom_version_get_test.go

```diff
@@ -0,0 +1,30 @@
+package k8s
+
+import (
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+)
+
+////
+// Simple use cases
+////
+func Test_GetVersion(t *testing.T) {
+	t.Run("simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw k8s version get 1.17.4",
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
+			core.TestCheckExitCode(0),
+		),
+	}))
+
+	t.Run("error", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw k8s version get test",
+		Check: core.TestCheckCombine(
+			core.TestCheckGolden(),
```

Comment:
should use `TestCheckError` as we don't want to test presentation

easier to have everything in the same place than to check in a different `.golden` file